packages/harmony/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "The Audius Design System",
   "license": "ISC",
   "author": "Audius",
-  "main": "dist/index.es.js",
-  "module": "dist/index.es.js",
-  "jsnext:main": "dist/index.es.js",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "jsnext:main": "dist/index.js",
```

Comment:
oh interesting. went with the es modules at first, but does this fix issues that we were having?

Reply:
so interestingly, after upgrading rollup, it asked to specify a build directory rather than file name, and then it did some interesting things breaking out every module into a separate file rather than one big one, which apparently can help treeshaking. one result of this is that it defaults to dist/index.js for the esm file, so that's why this was updated! the more im thinking about it though, if we want jest to work on web with harmony components we may also need to ship with a commonjs entry, but we can address that if necessary later.